src/fits.jl

```diff
@@ -0,0 +1,153 @@
+# helper functions
+
+# Gets the data from hdu
```

Comment:
not sure if this is an old or new comment, but I'd specify _why_ this function exists- mention how the data is read transposed from FITSIO.jl